modules/gds_ssh_config/files/gds_ssh_config

```diff
@@ -51,3 +51,17 @@ Host jumpbox-2.management.production
 
 Host *.production
   ProxyCommand ssh -e none %r@jumpbox-1.management.production -W $(echo %h | sed 's/\.production$//'):%p
+
+
+# Carrenza Migration ( Staging + Production )
+Host jumpbox.staging.publishing.service.gov.uk
+  ProxyCommand none
+
+Host *.staging.publishing.service.gov.uk
+  ProxyCommand ssh -e none %r@jumpbox.staging.publishing.service.gov.uk -W %h:%p
```

Comment:
SSH for that works through the new jumpbox! I was amazed the first time that worked :tada: